newton/_src/solvers/implicit_mpm/solve_rheology.py

```
@@ -95,50 +98,79 @@ def compute_delassus_diagonal(
+
+    local_strain_rhs[tau_i] = ev * strain_rhs[tau_i]
+    local_stress[tau_i] = wp.cw_mul(ev * stress[tau_i], diag)
+
+    for b in range(block_beg, block_end):
+        local_strain_mat_values[b] = ev * strain_mat_values[b]
+
+
+@wp.kernel
+def project_initial_stress(
+    stress: wp.array(dtype=vec6),
+    node_volume: wp.array(dtype=float),
```

Ruff (0.12.2)

Unused function argument: `node_volume`

(ARG001)